src/app/repositories/user.ts

```diff
@@ -10,10 +10,19 @@ class UserRepository implements IUserRepository {
     this.#userModel = model;
```

Comment:
If you are using TS, it is better to use its syntax for access modifiers. https://www.typescriptlang.org/docs/handbook/2/classes.html#private

Reply:
fixed